projects/igniteui-angular/src/lib/core/styles/components/chip/_chip-theme.scss

```
@@ -395,7 +402,7 @@
 
     %igx-chip__remove {
         display: inline-flex;
-        color: currentColor;
+        color: --var($theme, 'remove-icon-color');
```

Since the schema value is currentColor, we should provide a fallback, like so:
```suggestion
        color: --var($theme, 'remove-icon-color', currentColor);
```